project.mml

```diff
@@ -1499,7 +1501,7 @@ Layer:
                                                   'police', 'post_box', 'post_office', 'pub', 'biergarten', 'recycling', 'restaurant', 'food_court',
                                                   'fast_food', 'telephone', 'taxi', 'theatre', 'toilets', 'drinking_water',
                                                   'prison', 'hunting_stand', 'nightclub', 'veterinary', 'social_facility',
-                                                  'charging_station', 'arts_centre', 'ferry_terminal') THEN amenity ELSE NULL END,
+                                                  'charging_station', 'arts_centre', 'ferry_terminal', 'parking_entrance') THEN amenity ELSE NULL END,
```

Comment:
Are there other cases where we've added something to this query, but not the one above?

Reply:
You mean `amenity-points` but not `amenity-points-poly`? I don't know, I just saw on the wiki that it's allowed only on nodes. I'm still undecided however if should I add it to `amenity-line`, since it makes sense to tag the service highway.

What's your opinion?